src/process_model_output.R

```diff
@@ -5,6 +5,7 @@ library(dplyr)
 
 args <- commandArgs(trailingOnly=TRUE)
 today <- args[1] #today <- "2019-07-31"
+validate_data <- args[2] == "yes" # defaults to 'yes'
```

Comment:
any reason this wouldn't just be `args[2]` ? instead of `args[2] == "yes"` ? wondering if this would override an incoming 'no'

Reply:
This changes the string `'yes'` or `'no'` into a boolean, which is needed for the `if` statement further below. So, if that arg is `'no'`, then this statement returns `FALSE`

Reply:
ah! gotcha!